skgstat/DirectionalVariogram.py

```diff
@@ -550,6 +569,8 @@ def search_area(self, poi=0, ax=None):
         plot
 
         """
+        assert False
+        
```

Comment:
Same here, is the `search_area` function meant to be removed?

Reply:
Yep... Or, it's used to plot the search area, so I'm unsure if we maybe should replace it with something else? I added a function to plot point pairs, maybe that's good enough for debugging?